.github/workflows/tests.yml

```diff
@@ -0,0 +1,193 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  linux_tests:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        php: ["5.6", "7.0", "7.1", "7.2", "7.3", "7.4"]
+        stability: ["Stable"]
+        include:
+          - php: "5.6"
+            stability: "Lowest"
+          - php: "7.4"
+            stability: "Lowest"
+          - php: "8.0"
+            stability: "Ignore"
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }} - Linux
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer:v2
+          coverage: none
+
+      - name: Setup problem matchers
+        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+      - name: Install platform ignoring dependencies
+        uses: nick-invision/retry@v1
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
+          command: composer update --prefer-dist --no-interaction --no-progress --ignore-platform-req=php
```

Comment:
I would remove `--no-progress` so we can see what happens if needed.

Is `--no-interaction` required at all?

Reply:
No progress doesn't "hide progress" it hides the dynamic "process bars". The steps are still shown.

Reply:
True. What about `--no-interaction`? Can we remove it?